app/locales/de/messages.po

```diff
@@ -892,9 +892,14 @@ msgstr "Sie können diese Visualisierung teilen oder sie einbetten. Zudem könne
 
 #: app/components/form.tsx
 #: app/components/hint.tsx
+#: app/components/hint.tsx
 msgid "hint.loading.data"
 msgstr "Lade Daten …"
 
+#: app/components/hint.tsx
+msgid "hint.loading.data.large.datasets"
+msgstr "Kann bei großen Datensätzen mehr als 30 Sekunden dauern."
```

Comment:
@AnninaWalker Could you take a quick look ? The original sentence is "It may take more than 30 seconds for large datasets.".